config/modules/solanum/default.nix

```diff
@@ -232,6 +234,81 @@ in {
         };
       }));
     };
+
+    auth = mkOption {
+      description = ''
+        List of attribute sets defining auth blocks (I-Line), ordered by precedence.
+      '';
+      type = listOf (submodule {
+        options = {
+          hostmasks = mkOption {
+            type = listOf str;
+            description = ''
+              List of hostmasks the users is allowed to connect from.
+            '';
+          };
+
+          class = mkOption {
+            type = enum (builtins.attrNames cfg.classes);
```

Comment:
Like this :+1:

Reply:
Yep, just not too great for potential documentation.

Reply:
Alternative would be an assertion. But we can fix this later, if needed